packages/worker/src/connection.ts

```diff
@@ -1,29 +1,208 @@
 import util from 'node:util';
+import { AsyncLocalStorage } from 'node:async_hooks';
+import * as grpc from '@grpc/grpc-js';
+import { getRelativeTimeout } from '@grpc/grpc-js/build/src/deadline';
```

Comment:
That function is pretty simple. I'd prefer we inline it here rather than have an import to their internal project structure.

Reply:
Will do.